server/area_manager.py

```diff
@@ -766,25 +771,21 @@ def get_area_by_name(self, name: str) -> Area:
 
     def get_area_by_id(self, area_id: int) -> Area:
         """Get an area by ID
-
         Args:
             area_id (int): The ID of the area you are looking for
-
         Raises:
             AreaError: The Area ID does not exist
-
         Returns:
             Area: The Area
         """
 
         for area in self.areas:
-            if area.id == area_id:
+            if area.id == num:
```

Comment:
?

Reply:
woops, that's not supposed to be like that